module/gurps.js

```diff
@@ -2238,20 +2239,20 @@ if (!globalThis.GURPS) {
 
     game.settings.set(Settings.SYSTEM_NAME, Settings.SETTING_CHANGELOG_VERSION, GURPS.currentVersion.toString())
 
-    // get all aliases defined in the resource tracker templates and register them as damage types
-    let resourceTrackers = ResourceTrackerManager.getAllTemplates()
-      .filter(it => !!it.tracker.isDamageType)
-      .filter(it => !!it.tracker.alias)
-      .map(it => it.tracker)
-    resourceTrackers.forEach(it => (GURPS.DamageTables.damageTypeMap[it.alias] = it.alias))
-    resourceTrackers.forEach(
-      it =>
-        (GURPS.DamageTables.woundModifiers[it.alias] = {
-          multiplier: 1,
-          label: it.name,
-          resource: true,
-        })
-    )
+    // // get all aliases defined in the resource tracker templates and register them as damage types
```

Comment:
If this is no longer needed then it should be deleted rather than commented out.